modules/templates/helper.go

```diff
@@ -278,6 +281,41 @@ func renderCommitMessage(msg string, opts markup.RenderIssueIndexPatternOptions)
 	return template.HTML(msgLines[0])
 }
 
+func RenderCommitBody(msg, urlPrefix string, metas map[string]string) template.HTML {
+	return renderCommitBody(msg, markup.RenderIssueIndexPatternOptions{
+		URLPrefix: 	urlPrefix,
+		Metas:		metas,
+	})
+}
+
+func RenderCommitBodyLink(msg, urlPrefix string, urlDefault string, metas map[string]string) template.HTML {
+	return renderCommitMessage(msg, markup.RenderIssueIndexPatternOptions{
+		DefaultURL: urlDefault,
+		URLPrefix:  urlPrefix,
+		Metas:      metas,
+	})
+}
+
+func renderCommitBody(msg string, opts markup.RenderIssueIndexPatternOptions) template.HTML {
+	cleanMsg := template.HTMLEscapeString(msg)
+	fullMessage := string(markup.RenderIssueIndexPattern([]byte(cleanMsg), opts))
+	msgLines := strings.Split(strings.TrimSpace(fullMessage), "\n")
+	if len(msgLines) == 0 {
+		return template.HTML("")
+	}
+	body := msgLines[1:]
+	return template.HTML(strings.Join(body, "\n"))
+}
+
+func IsMultilineCommitMessage(msg string) bool {
+	msgLines := strings.Split(strings.TrimSpace(msg), "\n")
```

Comment:
just do
```
return strings.Count(strings.TrimSpace(msg), "\n") > 1
```

Reply:
Sweet, I was looking for a better way to do that.